metricbeat/module/kibana/_meta/docs.asciidoc

```diff
@@ -1,4 +1,4 @@
 == kibana Module
 
-This is the kibana Module.
+This is the kibana Module and only tracks the high level metrics. For more detailed monitoring of kibana use the https://www.elastic.co/guide/en/x-pack/current/xpack-monitoring.html[free x-pack monitoring].
```

Comment:
May I suggest saying something like "To monitor Kibana Elasticsearch metrics, use our https://www.elastic.co/guide/en/x-pack/current/xpack-monitoring.html[X-Pack monitoring] which is available under a free basic license."